apps/fluent-tester/src/E2E/common/consts.ts

```diff
@@ -0,0 +1,15 @@
+/* Accessibility Role Constants - https://microsoft.github.io/windows-docs-rs/doc/bindings/Windows/Win32/UI/Accessibility/ */
```

Comment:
The link here doesn't resolve for me. Where is it from?

Reply:
Whoops, missed the ending of the link. I will update in my newest commit